toponetx/generators/random_cell_complexes.py

```diff
@@ -0,0 +1,34 @@
+"""Generators for random cell complexes."""
+
+import networkx as nx
+
+from toponetx.classes import CellComplex
+from toponetx.transform.graph_to_cell_complex import homology_cycle_cell_complex
+
+__all__ = [
+    "cc_np_complex",
+]
+
+
+def cc_np_complex(n: int, p: float, seed=None) -> CellComplex:
```

Comment:
seed=None is invalid default condition as it will raise an exception, when the user doesn't provide a seed.